requirements.txt

```diff
@@ -12,3 +12,5 @@ tqdm  # comes with 3rd-party dependency
 rouge-score>=0.0.4
 sentencepiece>=0.1.95
 pytorch-lightning-bolts==0.3.0
+filelock  # comes with 3rd-party dependency
+pycocotools>=2.0.2 ; python_version >= "3.7"
```

Comment:
Python 3.6 users cant do detection?

Reply:
I guess it is limited by `pycocotools` on pip packages.

Reply:
That's strange, since 
1.) I have used pycocotools with older python versions as well
2.) The repos [setup.py](https://github.com/cocodataset/cocoapi/blob/master/PythonAPI/setup.py) doesn't say anything about this requirement (and in fact, I don't see any 3.7 specific code here)

Reply:
Yup, it does work for older python versions. Need to add the `cython` & `numpy` dependencies. [Issue1](https://github.com/cocodataset/cocoapi/issues/256), [Issue2](https://github.com/cocodataset/cocoapi/issues/468). Detectron2 corresponds to this [pycocotools](https://github.com/ppwwyyxx/cocoapi) installation with some updates.

Reply:
Hi @justusschock , `pycocotools` on pip corresponds to <https://github.com/ppwwyyxx/cocoapi>, they do not support versions less than python3.7 if you are using `pip install pycocotools` to install `pycocotools`. Other methods are available if you are using low Python version.

Reply:
For the recent release, this seems to be true due to more recent numpy features being used.

However, if this restricts the Python version, we should move down to `pycocotools==2.0.0` to resolve this restriction.

Reply:
Haha, I had a hunch we will be discussing `pycocotools` installation, which happens a lot.

@justusschock Would like your opinion on [pycocotools-fix](https://github.com/junjuew/cocoapi), it will solve the dependency constraint. What do you think?

Reply:
I think for starters, it's good. As we already have a lot of dependencies in Flash. I believe for the older python versions, we could add a brief `pycocotools` installation process in Docs.

Reply:
As Cython is throwing errors for certain systems & py versions.

Reply:
This would be fine for lightning, but with flash which also has a core audience for starters, I think we should aim for easy install and out of he box usage, which is why we definitely should look into a solution working on all OS and with all Python versions > 3.5

I also don't know why cython would throw errors. I just tried with a 3.5 and installed it by pip without any Problems. And since cython is kind of provided by python itself, I also would expect this behaviour for all other versions as well...

Where did you encounter issues?